src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -66,7 +66,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('collectors.xml');
         $loader->load('guard.xml');
 
-        if ($container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug')) {
+        if ($container->getParameter('kernel.debug')) {
             $loader->load('security_debug.xml');
```

Comment:
well, if you do it, the security panel would not appear when enabling the profiler in no-debug mode, which might be confusing